README.md

```diff
@@ -79,6 +79,9 @@ correctly.
 pebble -config ./test/config/pebble-config.json
 ```
 
+Afterwards you can access the Pebble server's ACME directory
+at `http://localhost:14000/dir`.
```

Comment:
at `https://localhost:14000/dir` ...

Reply:
:man_facepalming: Thanks for catching that @shred ! Fixed in https://github.com/letsencrypt/pebble/pull/170